app/models/ability.rb

```diff
@@ -3,7 +3,10 @@ class Ability
 
   def initialize(user)
     if user.class == Admin
-      can :read, Admin, id: user.id
+      can [:read, :update, :destroy], Admin, id: user.id
+      can [:new, :create, :destroy], Admin
+      can [:read, :edit, :update, :new, :create, :destroy], Item
+      can [:edit, :update, :new, :create, :destroy], Category
     elsif user.class == User
```

Comment:
should we include :read with Category? and for Item and Category could we just put ":all Item" and ":all Category"? Instead of listing out all of the actions?